plugins/template/src/lib/Template.cpp

```diff
@@ -6,48 +6,28 @@
 #define BUILD_VERSION "TestBuild"
 #endif
 
+#include "Template.h"
```

Comment:
nit: Include order

Reply:
What do you mean? Includes are sorted automatically.